databuilder/databuilder/extractor/openlineage_extractor.py

```diff
@@ -0,0 +1,93 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import logging
+from typing import Any, Dict, Iterator
+
+from pyhocon import ConfigTree
+
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.models.table_lineage import TableLineage
+
+LOGGER = logging.getLogger(__name__)
+
+
+class OpenLineageTableLineageExtractor(Extractor):
+    # Config keys
+    TABLE_LINEAGE_FILE_LOCATION = 'table_lineage_file_location'
+    CLUSTER_NAME = 'cluster_name'
+    OL_INPUTS_KEY = 'inputs_key'
```

Comment:
can you describe (in README.md) what those `OL_` settings mean?

Reply:
done